pkg/sql/schemachanger/scbuild/internal/scbuildstmt/dependencies.go

```diff
@@ -75,6 +75,10 @@ type BuilderState interface {
 	// Ensure ensures the presence of the given element in the BuilderState with
 	// the given statuses and metadata.
 	Ensure(current scpb.Status, target scpb.TargetStatus, elem scpb.Element, meta scpb.TargetMetadata)
+
+	// CheckNoConcurrentSchemaChange checks no concurrent schema change exists for
+	// a given descriptor ID.
+	CheckNoConcurrentSchemaChange(descID catid.DescID)
```

Comment:
Given we call this `scbuild`, can we remove it from the interface? Do we need any calls to it in `scbuildstmt`?